core-bundle/contao/languages/en/tl_user.xlf

```diff
@@ -68,6 +68,12 @@
       <trans-unit id="tl_user.doNotCollapse.1">
         <source>Do not collapse content elements in the back end preview.</source>
       </trans-unit>
+      <trans-unit id="tl_user.doNotHideMessages.0">
+        <source>Do not auto-hide flash messages</source>
+      </trans-unit>
+      <trans-unit id="tl_user.doNotHideMessages.1">
+        <source>Keep flash messages open until I close them.</source>
+      </trans-unit>
```

Comment:
```suggestion
      <trans-unit id="tl_user.doNotHideMessages.0">
        <source>Keep pop-up messages</source>
      </trans-unit>
      <trans-unit id="tl_user.doNotHideMessages.1">
        <source>Do not automatically hide pop-up messages.</source>
      </trans-unit>
```

I think "pop-up messages" is more understandable for the average user than "flash messages".